plugins/woocommerce/src/Internal/StockNotifications/Admin/SettingsController.php

```diff
@@ -106,7 +106,7 @@ public function add_customer_stock_notifications_settings( $settings, $section_i
 				array(
 					'title'   => __( 'Delete unverified notification sign-ups after (in days)', 'woocommerce' ),
 					'desc'    => __( 'Controls how long the plugin will store unverified notification sign-ups in the database. Enter zero, or leave this field empty if you would like to store expired sign-up requests indefinitey.', 'woocommerce' ),
-					'id'      => 'wc_customer_stock_notifications_unverified_deletions_days_threshold',
+					'id'      => 'woocommerce_customer_stock_notifications_unverified_deletions_days_threshold',
 					'default' => Config::get_unverified_deletion_days_threshold(),
```

Comment:
Renaming `wc_customer_stock_notifications_unverified_deletions_days_threshold` => `woocommerce_customer_stock_notifications_unverified_deletions_days_threshold` for consistency. This'll show a few more times in this changeset. Sorry for the noise.